authentication/account/repository/user.go

```diff
@@ -174,7 +174,7 @@ func (m *GormUserRepository) Delete(ctx context.Context, id uuid.UUID) error {
 }
 
 // List return all users
-func (m *GormUserRepository) List(ctx context.Context) ([]User, error) {
+func (m *GormUserRepository) List(ctx context.Context, funcs ...func(*gorm.DB) *gorm.DB) ([]User, error) {
```

Comment:
It seems that you don't use funcs in this method. Then why adding it?

Reply:
Good Catch. Addressed in 72b2576